index.html

```diff
@@ -2,7 +2,130 @@
 <html>
   <head>
     <title>Memory exercise</title>
+    <style>
+	table, th, td {
+	   border: 1px solid black;
+	   color: white;
+	   padding: 5px;
+	}
+	</style>
+	<script src="jquery-2.1.4.min.js"></script>
   </head>
   <body>
+  <div id="board"></div>
+
+<script>
+
+
+// GLOBAL VARIABLES TO REMEMBER THE FIRST CARD FLIP
+var c = 0;
+var firstval;
+
+
+var Board = function (rows, cols) {
+
+	if ((rows*cols)%2 > 0) { alert('Not an even number'); }
+
+	$('#board').append('<table id=tid></table>');
+	
+	var rowString = "";
+	// CREATES BOARD OF ROWS x COLS
+	for (var i=0; i<rows; i++) {
+		rowString="<tr>";
+		for (var j=0; j<cols; j++) {
+			rowString+="<td>.</td>";
+		}
+		rowString+="</tr>";
+		$('#board').children().append(rowString);
+	}
+}
+
+Board.prototype.setBoard = function() {
+
+	var table = document.getElementById('tid');
+	var cells = table.getElementsByTagName('td');
+	var values = new Array();
+	var pairValues = new Array();
```

Comment:
Minor: I would use array literals here instead (`[]`), unless you have a specific reason to need the constructor form.